src/preamble.js

```diff
@@ -850,6 +850,9 @@ if (!isDataURI(wasmBinaryFile)) {
 function getBinary() {
   try {
     if (Module['wasmBinary']) {
+      if (Module['wasmBinary'] === true) {
```

Comment:
Is there a problem with doing `new Uint8Array(true)`? I guess that throws on IE11?

How about changing the original `Module['wasmBinary'] = true;` to assign `[]` or `[0]`? Seems less surprising than the change here, which would need an explanation.

Reply:
Yeah, `new Uint8Array(true)` throws on IE11. Using `[]` should work though -- will give us an empty binary, won't throw passed into the TypedArray constructor, and will still evaluate true-ish. :)